lib/crypto.js

```diff
@@ -93,6 +93,14 @@ class Crypto {
   generateSigningKeypair() {
     this.__notImplemented('generateSigningKeypair')
   }
+
+  signDetached() {
```

Comment:
What are these two for?  Could there be a comment here, since there's no code yet?

Reply:
I followed the convention of these methods to not have comments here but rather in the sodium sdk where they're implemented; Do you think they should have comments here also?

Reply:
![](https://i.imgflip.com/362csr.jpg)